All Core Devs Meetings/Meeting 95.md

```diff
@@ -0,0 +1,606 @@
+
+This EIP may go into YOLOv2.
+
+Something to consider for YOLO version, clients have to implement all EIPs going into the testnet, or tney'll be kicked out. So, a lot of these EIPs going into YOLOv2 are very simple. This one is 2 lines of specification.
+
+## 1.7 EIP-2711: Sponsored, expiring and batch transactions.
+
+Video | [1:08:01](https://youtu.be/-Jefyrs4f70?t=4141)
+-|-
+
+This was supposed to be the first 2718 transaction. A new transaction type that handles user experiences annoyances in the Dapp space.
+
+Sponsored transactions: Instead of having a single signer, we have 2 signers. The second signer is the one who pays for gas. This is in layer 1, which is much cleaner than solving it in the Dapp layer.
+
+
+Batch transactions: One or more transactions, atomic in that they make it in the order you specify, with nothing in-between. They are not atomic in that they all fail or they all succeed. Instead of having Dapps that send multiple transactions, one to approve, and the other to commit the action, it's a single transaction.
+
+Also adds optional expiration field, that invalidates a transaction after a specified timestamp. Uniswap has this currently. But, the transactions still exist on chain, so after their expiry in uniswap, it just fails instantly, which leads to large losses of gas.
+
+There's one gas price for the entire batch.
+
+Time boxed until later.
+
+
+## 1.8 EIP-1057 Next Steps.
+
+Video | [1:19:09](https://youtu.be/-Jefyrs4f70?t=4749)
+-|-
+
+Instead of going into decisions, just going into updates.
+
+Andrea, one of the authors, just got out of the hospital, so he's reviewing code just before he went into the hospital. It looks like the kik exploit is nothing to worry about.
+
+It looks like existing exploits will fall out of the network in November, so that's a time to deploy ProgPoW if it will be deployed.
+
+
+# 2. EIP & Upgrades Updates
+
+Video | [5:29](https://youtu.be/-Jefyrs4f70?t=329)
+-|-
+
+A lot of questions about Berlin and timing for Berlin. Why the timing for Berlin hasn't been announced is that it's dependent on YOLOv2. For EIPs that go into YOLOv2, it is possible they go into Berlin, but it doesn't mean they are slated for the Berlin upgrade.
+
+EIPs in YOLOv2 are considered for testnet first, then client integration testing, and that will determine its inclusion.
+
+As for the name, it's not called Berlinv2, as EIPs that are in YOLOv2 may not go into a hardfork at all. Testnets after Berlin may be YOLOv3, or something else. YOLO is meant to show that the testnet will live for a short time.
+
+## 2.1 YOLO / YOLOv2 & Berlin state tests update
+
+Video | [1:20:54](https://youtu.be/-Jefyrs4f70?t=4854)
+-|-
+
+Going into YOLOv2:
+- 2718
+- 2929
+- 2935
+- 2711
+
+Continue discussion on:
+- 2930
+- 2315 (already on YOLOv1)
+
+2 left to talk about: 2565. 2565 is moving into final. The other is 2046. 2929 would supercede 2046. 2046 is pending until 2929 is decided on.
```

Comment:
**James**: 1. that matches my notes.
                2. left to talk about is 2565.